mapie/regression/time_series_regression.py

```diff
@@ -9,30 +9,52 @@
 
 from mapie._typing import ArrayLike, NDArray
 from mapie.conformity_scores import ConformityScore
-from .regression import MapieRegressor
+from mapie.regression import MapieRegressor
+from mapie.utils import (check_alpha, check_gamma, convert_to_numpy)
 
 
 class MapieTimeSeriesRegressor(MapieRegressor):
     """
     Prediction intervals with out-of-fold residuals for time series.
+    This class only has two valid ``method`` : ``"enbpi"`` or ``"aci"``
 
-    This class implements the EnbPI strategy for estimating
-    prediction intervals on single-output time series. The only valid
-    ``method`` is ``"enbpi"``.
+    The prediction intervals are calibrated on a split of the trained data.
+    Both strategies are estimating prediction intervals
+    on single-output time series.
+
+    EnbPI allows you to update conformal scores using the ``partial_fit``
+    function. It will replace the oldest one with the newest scores.
+    It will keep the same amount of total scores
 
     Actually, EnbPI only corresponds to ``MapieTimeSeriesRegressor`` if the
     ``cv`` argument is of type ``BlockBootstrap``.
 
+    The ACI strategy allows you to adapt the conformal inference
+    (i.e the quantile). If the real values are not in the coverage,
+    the size of the intervals will grow.
+    Conversely, if the real values are in the coverage,
+    the size of the intervals will decrease.
+    You can use a gamma coefficient to adjust the strength of the correction.
```

Comment:
Maybe add that in some cases you can have infinite set size

Reply:
> The ACI strategy allows you to adapt the conformal inference
>     (i.e the quantile). If the real values are not in the coverage,
>     the size of the intervals will grow. Conversely, if the real values are in
>     the coverage, the size of the intervals will decrease. You can use a gamma
>     coefficient to adjust the strength of the correction. If the quantile is
>     set to zero, the method will produce an infinite set size.

Reply:
Maybe not "is set" because it's not the user who is choosing this value, prefer is equal

Reply:
I'm putting that aside because in practice, that's not what's being done.

Reply:
I open an issue to allow the production of infinite intervals and I link it to this PR.

Reply:
Done.